### ISSUE TITLE
CTC loss is not getting below 36% and word error is more than 600%. can anyone help in that?

### ISSUE THREAD

**Issue by shefalilodha** [sushant097/Handwritten-Line-Text-Recognition-using-Deep-Learning-with-Tensorflow#4]

**Comment by shefalilodha** [comment]
![image](https://user-images.githubusercontent.com/34569963/70415940-dbddb700-1a83-11ea-8839-fa7b57bf2907.png)

**Comment by rpro91** [comment]
WER cant be this much high.. Can you please elaborate how you running this model

**Comment by shefalilodha** [comment]
> WER cant be this much high.. Can you please elaborate how you running this model

as mentioned in the ReadMe, I have put iam dataset lines.txt and and lines folder into the data folder . then training it form command line using the command "python main.py --train"

**Comment by rpro91** [comment]
I have trained this model on my end and WER was not too high. 
Still if you want to work on Line level HTR you can go for below one also
https://github.com/lamhoangtung/LineHTR

**Comment by shefalilodha** [comment]
> model on my end and WER was not too high.
> Still if you want to work on Line level HTR you can go for below one also

can you tell me how much time it took to train on your system and what was your system configuration?
Thanks

**Comment by rpro91** [comment]
config- 16GB RAM, i5 processor.. it took around 37 hours

**Comment by shefalilodha** [comment]
> 37 hours

can you share your pre-trained model ,if possible.

**Comment by mcmalzahar** [comment]
I'm also getting incredibly poor scores, very similar to those that the original poster had.
Using the IAM lines dataset, all according to instructions.
Not using word beam search decoding or anything similar, just `python main.py --train`
Exact same results using either CPU or GPU (the only difference of course being that the GPU way is several times faster). The CTC loss scores and the error rate stop improving after the 7th or so epoch.
Tried increasing the file input size, but got considerably worse results.
Attached an image of an example word error rate while training.

![screenshot_1](https://user-images.githubusercontent.com/25721361/80967357-eaf04780-8e1e-11ea-9426-237bce64ca6f.png)

My specs are:
16 GB RAM, Ryzen 3700X CPU, NVIDIA RTX2070 SUPER GPU. Using Docker and TensorFlow 1.13.
